microreactors/gcmr/core/Neutronics/Griffin_steady_state.i

```diff
@@ -0,0 +1,164 @@
+######################################################################################################
+## Griffin steady state Model of whole-core Gas-cooled Microreactor
+# If using or referring to this model, please cite as explained in
+# https://mooseframework.inl.gov/virtual_test_bed/citing.html
+#####################################################################################################
+
+[Mesh]
+  [fmg]
+    type = FileMeshGenerator
+    file = '../MESH/Griffin_mesh_in.e'
+  []
+
+  [fmg_id]
+    type = SubdomainExtraElementIDGenerator
+    input = fmg
+    subdomains = '10 100 101 102 103 200 201 400 401 4000 4001 40000 40001 300 301 600 602 603 604 1000 1003 19000 29000 39000 49000 59000 19003 29003 39003 49003 59003 19900 29900 39900 49900 59900 19903 29903 39903 49903 59903 1777 1773 250'
+    extra_element_id_names = 'material_id'
+    extra_element_ids = '803 802 802 810 806 807 807 700 700 701 701 702 702 807 807 811 811 809 807 805 805 764 764 764 764 764 764 764 764 764 764 754 754 754 754 754 754 754 754 754 754 809 809 811'
+  []
+
+  [coarse_mesh]
+    type = GeneratedMeshGenerator
+    dim = 3
+    nx = 20
+    ny = 20
+    nz = 20
+    xmin = -1.21
+    xmax = 1.21
+    ymin = -1.21
+    ymax = 1.21
+    zmin = 0.0
+    zmax = 2.4
+  []
+
+  [assign_coarse_id]
+    type = CoarseMeshExtraElementIDGenerator
+    input = fmg_id
+    coarse_mesh = coarse_mesh
+    extra_element_id_name = coarse_element_id
+  []
+[]
+
+[Executioner]
+  type = SweepUpdate
+  richardson_abs_tol = 1e-8
```

Comment:
I would re-order the parameters and group them (with a comment) for readability.
maybe 2 groups?
- richardson iterations
- CMFD transport acceletation

Reply:
Done.